### ISSUE TITLE
Empty XML Tag errors

### ISSUE THREAD

**Issue by iNoles** [Tickaroo/tikxml#33]
I can't control remote  xml files.

When XML Tag is like 
```xml
<empty />
```
I get error message which is Expected xml element text content but was ELEMENT_END.

Why not to leave a blank text on it?

**Comment by sockeqwe** [comment]
Thanks for reporting, seems to be a bug ...

**Comment by sockeqwe** [comment]
Hm, I'm not able to reproduce this ... 
See Tests added: https://github.com/Tickaroo/tikxml/commit/15fd0163aa852e97c45a3b8ca4ffcdac260a2d29

Could you provide your annotated classes or even better a Unit Test with annotated classes and xml you are going to parse that shows that this is failing?

**Comment by iNoles** [comment]
When I do like
```xml
<example>
    <empty />
</example>
```
Java model class would be like
```java
@Xml
public class Example {
    @PropertyElement
    String empty;
}

**Comment by sockeqwe** [comment]
I see your problem, obviously in such a case the `<empty />` tag shouldn't appear at all in your xml file.
Nevertheless, what would be the expected result then? `null` or `emptyString`? I guess `null`? But if it's null, how do we represent an empty string? 

In general, should we simply do nothing in such a case so that the default value will be remain (i.e. `null` for objects like Strings, `0` for integers, `false` for booleans etc.?

note to myself: also happens when:

```xml
<example>
    <empty></empty>
</example>
```

**Comment by iNoles** [comment]
Set default values would be best to handle this situations.

**Comment by FabianTerhorst** [comment]
The xml pull parser has also a bug when you give him something like ```<empty />```. He is using the value from the next tag then.

**Comment by sockeqwe** [comment]
Parser returns now the default values like `""` (empty String) for empty elements like `<element></element>`. For booleans `false` will be returned, for numbers `0`.

If you want to model `null` values, simply don't write a empty tag in your xml file.

This fix is available in `0.5.3-SNAPSHOT`
